tests/core/training/story_reader/test_yaml_story_reader.py

```diff
@@ -119,3 +118,32 @@ async def test_yaml_wrong_yaml_format_warning(default_domain: Domain):
             tracker_limit=1000,
             remove_duplicates=False,
         )
+
+
+async def test_parsing_of_e2e_stories(default_domain: Domain):
+    yaml_file = "data/test_yaml_stories/stories_hybrid_e2e.yml"
+    tracker = await training.load_data(
+        yaml_file,
+        default_domain,
+        use_story_concatenation=False,
+        tracker_limit=1000,
+        remove_duplicates=False,
+    )
+
+    assert len(tracker) == 1
+
+    assert list(tracker[0].events) == [
+        ActionExecuted(ACTION_LISTEN_NAME),
+        UserUttered("simple", {"name": "simple"}),
```

Comment:
@evgeniiaraz Tiny problem: Currently we do the following for stories training data: intents become `UserUttered` events with the labeled intent and `text` == `intent`. This makes them a bit hard to distinguish them from "e2e" user utterances. As you can see in line 140 I temporarily solved this by giving "e2e" `UserUtterance` events a `text` but no `intent` attribute. Wdyt?

Reply:
The real text definitely shouldn't get into the `intent` field. So, yes, that's the way to go!
One thought here: do you think it would be better to set the `name` to `None`? (mostly for consistency)

Reply:
yep, will do that 👍